docs/adr/0010-common-auth-for-applications.md

```diff
@@ -0,0 +1,93 @@
+# A common authentication for applications
+
+- Status: accepted
+- Deciders: @tumido, @HumairAK, @anishasthana, @4n4nd
+- Date: 2021-01-19
+
+Technical Story: [description | ticket/issue URL] <!-- optional -->
```

Comment:
All these [issues here](https://github.com/operate-first/apps/issues?q=is%3Aissue+is%3Aopen+label%3Aauthentication) are part of the technical story no?

Reply:
Yes, I'll add them, thanks. As you can see, this bit kinda slipped my attention. :smile: